libvirt/tests/src/libvirt_mem.py

```
@@ -568,6 +708,20 @@ def modify_domain_xml():
+                    # Verify attach operation and store state for detach baseline
+                    if attach_option and baseline_xml_active is not None:
+                        try:
+                            after_attach_xml_active = get_domain_memory_xml(virsh, vm_name, check_inactive=False)
+                            after_attach_xml_inactive = get_domain_memory_xml(virsh, vm_name, check_inactive=True)
+                            verify_immediate_changes(attach_option, baseline_xml_active, after_attach_xml_active,
+                                                   baseline_xml_inactive, after_attach_xml_inactive,
+                                                   operation="attach", test=test)
+                            logging.info("After attach: active_xml=%s devices, inactive_xml=%s devices (will be reused as before-detach baseline)",
+                                        after_attach_xml_active.get('memory_device_count') if after_attach_xml_active else 'N/A',
+                                        after_attach_xml_inactive.get('memory_device_count') if after_attach_xml_inactive else 'N/A')
+                        except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)